src/CatalogPage.js

```diff
@@ -1,10 +1,53 @@
+import { it } from "node:test";
+
 /**
  * This class obtains the elements in the DOM for the
  * catalog page using different locators
  */
 class CatalogPage {
   constructor(page) {
     this.productTitle = page.getByTestId("title");
+    this.productPrices = page.getByTestId("inventory-item-price");
+    this.dropdown = page.locator("//select[@data-test='product-sort-container']");
   }
+
+  async getPriceListArray(){
+    const priceItemsArray = [];
+    for(let item of await this.productPrices.all()){
+      const price  = await item.innerText();
+      const priceFloat = parseFloat(price.replace("$", ""));
+      priceItemsArray.push(priceFloat);
+    }
+    return priceItemsArray;
+  }
+
+  async orderArrayAscending(itemsPriceArray){
+    var pricceItemOrdered = itemsPriceArray.sort(function (a,b) {
```

Comment:
`var` is used when you are declaring a global variable, use `let` instead to declare scoped variables.

If you make this method generic, `pricceItemOrdered` would need to be renamed to something more appropriate such as orderedList or something like that. Notice you have a typo `pricce`.